config/configuration.rb

```diff
@@ -0,0 +1,17 @@
+# dashboard app specific configuration
+class Configuration
+  class << self
+    attr_writer :app_development_enabled
+    attr_writer :app_sharing_enabled
+
+    def app_development_enabled?
+      return @app_development_enabled if defined? @app_development_enabled
+      @app_development_enabled = ENV['OOD_APP_DEVELOPMENT'].present?
```

Comment:
Another option is setting this in an initializer to maintain backwards compatibility for now:

```ruby
# config/initializers/read_env_vars.rb

Configuration.app_development_enabled = ENV["OOD_APP_DEVELOPMENT"].present?
Configuration.app_sharing_enabled = ENV["OOD_APP_SHARING"].present?
```

This will make it easier to remove in the future when it is deprecated.

Reply:
So the above method would just be...

```ruby
def app_development_enabled?
  @app_development_enabled
end
```

Reply:
Although might want to rename that initializer to `01_read_env_vars.rb` so it is loaded earlier than all the other initializers.

Or patch it directly into `config/environment.rb` so it loads before the initializers.

Reply:
> This will make it easier to remove in the future when it is deprecated.

That assumes we will remove it.

Reply:
I am fine with it as is. Didn't know if we slowly wanted to deprecate `dotenv` in favor of only a configuration file as it would simplify code and documentation.